src/components/Output/index.tsx

```diff
@@ -78,16 +77,16 @@ export default class Output extends Component<Props, State> {
 
   @bind
   zoomIn() {
-    this.setState({
-      scale: Math.min(this.state.scale * 1.25, 100),
-    });
+    if (!this.pinchZoomLeft) throw Error('Missing pinch-zoom element');
+
+    this.pinchZoomLeft.scaleTo(Math.min(this.state.scale * 1.25, 100), scaleToOpts);
```

Comment:
Worth removing the min/max here to let pinch-zoom decide?

Reply:
Agreed. Will do that.